packages/datagrid/src/components/datagrid.component.js

```diff
@@ -0,0 +1,246 @@
+			https://github.com/ag-grid/ag-grid/issues/2216
+			When Ag-grid implement this feature, we can't remove the below code
+		*/
+		// eslint-disable-next-line react/no-find-dom-node
+		const focusedCells = ReactDOM.findDOMNode(this.gridElement).querySelectorAll(
+			`.${FOCUSED_COLUMN_CLASS_NAME}`,
+		);
+
+		for (const focusedCell of focusedCells) {
+			focusedCell.classList.remove(FOCUSED_COLUMN_CLASS_NAME);
+		}
+	}
+
+	updateStyleFocusColumn() {
+		const colId = this.currentColId;
+
+		if (!colId || colId.includes(NAMESPACE_INDEX)) {
+			return;
+		}
+
+		/*
+			This is a bad pratice to manipulate straight the DOM.
+			But we can't the choice if we want hightlight the column.
+			There is a issue on ag-grid to request a feature like this.
+			https://github.com/ag-grid/ag-grid/issues/2216
+			When Ag-grid implement this feature, we can't remove the below code
+		*/
+		// eslint-disable-next-line react/no-find-dom-node
+		const columnsCells = ReactDOM.findDOMNode(this.gridElement).querySelectorAll(
+			`[col-id="${colId}"]:not(.${FOCUSED_COLUMN_CLASS_NAME})`,
+		);
+
+		for (const columnCell of columnsCells) {
+			columnCell.classList.add(FOCUSED_COLUMN_CLASS_NAME);
+		}
+	}
+
+	handleKeyboard({ nextCellDef, previousCellDef }) {
+		if (!nextCellDef) {
+			return null;
+		}
+
+		if (previousCellDef.rowIndex !== nextCellDef.rowIndex) {
+			if (this.gridAPI) {
+				this.gridAPI.getDisplayedRowAtIndex(nextCellDef.rowIndex).setSelected(true, true);
+			}
+		}
+
+		return nextCellDef;
+	}
+
+	render() {
+		const agGridOptions = {
+			headerHeight: this.props.headerHeight,
+			tabToNextCell: this.handleKeyboard,
+			navigateToNextCell: this.handleKeyboard,
+			onViewportChanged: this.updateStyleFocusColumn,
+			onVirtualColumnsChanged: this.updateStyleFocusColumn,
+			ref: this.setGridElement,
+			rowData: this.props.getRowDataFn(this.props.data),
+			rowHeight: this.props.rowHeight,
+			rowSelection: this.props.rowSelection,
+			suppressDragLeaveHidesColumns: true,
+			onCellFocused: this.onFocusedCell,
+			onGridReady: this.onGridReady,
+		};
+
+		const pinnedColumnDefs = this.props.getPinnedColumnDefsFn(this.props.data);
+		const columnDefs = this.props.getColumnDefsFn(this.props.data);
+
+		if (pinnedColumnDefs) {
+			agGridOptions.columnDefs = pinnedColumnDefs.map(pinnedColumnDefinition => ({
+				lockPosition: true,
+				pinned: 'left',
+				valueGetter: this.props.getValueGetterFn,
+				...pinnedColumnDefinition,
+				[AG_GRID_CUSTOM_HEADER_KEY]: PIN_HEADER_RENDERER_COMPONENT,
+			}));
+		}
+
+		if (columnDefs) {
+			const columnsDefs = columnDefs.map(columnDefinition => ({
+				lockPinned: true,
+				valueGetter: this.props.getValueGetterFn,
+				...columnDefinition,
+				[AG_GRID_CUSTOM_CELL_KEY]: CELL_RENDERER_COMPONENT,
+				[AG_GRID_CUSTOM_HEADER_KEY]: HEADER_RENDERER_COMPONENT,
+			}));
+			agGridOptions.columnDefs = [...agGridOptions.columnDefs, ...columnsDefs];
+		}
+
+		agGridOptions.frameworkComponents = {
```

Comment:
This block defines the (pinned-)header and cell renderer to use, but this makes `props.getComponent` mandatory.
If we just, want to use some default renderer, we have to define a getComponent that map the keys to the default renderers.
That's not really dev friendly.
In `Inject.get`, you can pass a 3rd argument, which is the default value to return if there is no getComponent.